scripts/connect_clouddb_proxy.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+####################################
+# connect_clouddb_proxy.py
+#
+# This is a simple wrapper that looks up
```

Comment:
I recall advocating for 80 column lines, but this is really short! ;-)

Reply:
Soon I'm going to push for 2-character lines! ;)